Packages/Tracking Preview/PhysicsHands/Runtime/PhysicsBone.cs

```diff
@@ -400,46 +448,103 @@ private bool RemoveGrabQueue(Rigidbody rigid)
             return true;
         }
 
-        private bool IsObjectGrabbable(HashSet<Collider> colliders)
-        {
-            foreach (var collider in colliders)
-            {
-                // Center and the base of the bone
-                if (Vector3.Dot(_grabDirectionCenter, (collider.ClosestPoint(_grabPositionCenter) - _grabPositionCenter).normalized) > 0.18f || Vector3.Dot(_grabDirection, (collider.ClosestPoint(_grabPositionBase) - _grabPositionBase).normalized) > 0.2f)
-                {
-                    return true;
-                }
-            }
-            return false;
-        }
+
 
         private void UpdateBoneHoverDistances()
         {
             _objectDistance = float.MaxValue;
 
             float tempDist;
-            Vector3 tempA, tempB, tempMid;
+            Vector3 colliderPos, bonePos, tempMid, direction;
```

Comment:
can we rename the "temp" variables to just distance and midPoint?

Reply:
sure :)